src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectAutoCRContainer.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+import { useAllLinkTypes } from '../../../hooks/projects.hooks';
+import { useToast } from '../../../hooks/toasts.hooks';
+import { useState } from 'react';
+import ProjectFormContainer from './ProjectForm';
+import { useHistory } from 'react-router-dom';
+import { routes } from '../../../utils/routes';
+import { getRequiredLinkTypeNames } from '../../../utils/link.utils';
+import ErrorPage from '../../ErrorPage';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import { CreateStandardChangeRequestPayload, useCreateStandardChangeRequest } from '../../../hooks/change-requests.hooks';
+import { ProjectProposedChangesCreateArgs, WbsElementStatus } from 'shared';
+import * as yup from 'yup';
+import { ProjectCreateChangeRequestFormInput } from './ProjectEditContainer';
+
+const CreateProjectCRContainer: React.FC = () => {
+  const toast = useToast();
+  const history = useHistory();
+
+  const [projectManagerId, setProjectManagerId] = useState<string | undefined>();
+  const [projectLeadId, setProjectLeadId] = useState<string | undefined>();
+
+  const { mutateAsync, isLoading } = useCreateStandardChangeRequest();
+  const {
+    data: allLinkTypes,
+    isLoading: allLinkTypesIsLoading,
+    isError: allLinkTypesIsError,
+    error: allLinkTypesError
+  } = useAllLinkTypes();
+
+  if (isLoading) return <LoadingIndicator />;
+  if (!allLinkTypes || allLinkTypesIsLoading) return <LoadingIndicator />;
+  if (allLinkTypesIsError) return <ErrorPage message={allLinkTypesError.message} />;
+
+  const requiredLinkTypeNames = getRequiredLinkTypeNames(allLinkTypes);
+
+  const defaultValues = {
+    name: '',
+    budget: 0,
+    summary: '',
+    teamIds: [],
+    carNumber: 0,
+    links: [],
+    crId: '',
+    goals: [],
+    features: [],
+    constraints: [],
+    rules: [],
+    projectLeadId,
+    projectManagerId
+  };
+
+  const schema = yup.object().shape({
+    name: yup.string().required('Name is required!'),
+    carNumber: yup.number().min(0).required('A car number is required!'),
```

Comment:
can you restrict this to 0-3 inclusive on all the forms for now, and add in a TODO (similar to the other one) to update it once we have the car model made.